### DISCUSSION TITLE
Cannot read container configuration. (Amazon S3) error on public folders

### DISCUSSION THREAD

**Discussion by igorschlum** [iterate-ch/cyberduck#16728] [Q&A] [answered]
When attempting to access S3 buckets through Cyberduck, the application incorrectly handles public vs private folders, leading to DNS lookup errors. 

<img width="851" alt="s3view" src="https://github.com/user-attachments/assets/240bf524-572a-427f-8eea-6b1a9c9da6ad" />

<img width="262" alt="errorCyberduck" src="https://github.com/user-attachments/assets/ec82f7d6-7268-490a-94be-bc892977351a" />

Specifically:

Public folders (e.g., agorapass) display incorrect URLs, causing DNS lookup failures.
Private folders (e.g., local_agorapass) are accessible without issues.
The error message displayed is: "Cannot read container configuration. (Amazon S3)"
DNS lookup for agorapass.storage.vicem.com failed. DNS is the network service that translates a server name to its Internet address. This error is most often caused by having no connection to the Internet or a misconfigured network. It can also be caused by an unresponsive DNS server or a firewall preventing access to the network.

My s3 is not responding to agorapass.storage.vicem.com as agorapass is a forder in storage.vicem.com

Environment
Operating System: [Mac OS 15.2
Cyberduck Version: [9.1.1]
S3 Provider: [storage.vicem.com]

Thank you for your attention to this matter. Any assistance in resolving this issue would be greatly appreciated.

[cyberduck.log.zip](https://github.com/user-attachments/files/18247329/cyberduck.log.zip)

**Comment by dkocher** [comment, marked as answer]
Please use the connection profile using deprecated path-style access for buckets as documented in [Connecting using Deprecated Path Style Requests](https://docs.cyberduck.io/protocols/s3/#connecting-using-deprecated-path-style-requests).

bq. Attempting to connect using the regular S3 connection profile to a server with no support for virtual host style requests will cause the error _Cannot read container configuration_.

**Comment by dkocher** [comment]
Also in #16590.

**Comment by igorschlum** [comment]
Thank you. it works well !
